.travis.yml

```diff
@@ -16,3 +20,9 @@ notifications:
     on_success: never
     on_failure: always
     on_start: never
+addons:
```

Comment:
can you just explain the usage of these tokens, and howto regenerate them and revoke them if necessary ?